Core.Collectors/Web/HttpClientWrapper.cs

```diff
@@ -16,6 +17,7 @@ public sealed class HttpClientWrapper : IHttpClient
         public static readonly TimeSpan HttpTimeout = TimeSpan.FromMinutes(10);
 
         private readonly HttpClient httpClient;
+        protected TelemetryClient TelemetryClient { get; }
```

Comment:
Why are we adding TelemetryClient here? 
This should also be private instead of protected because this is a sealed class (no inheritance allowed)

Reply:
Removed it as I dont require it with the new logic